lib/internal/errors.js

```diff
@@ -172,6 +172,10 @@ E('ERR_SOCKET_DGRAM_NOT_RUNNING', 'Not running');
 E('ERR_STDERR_CLOSE', 'process.stderr cannot be closed');
 E('ERR_STDOUT_CLOSE', 'process.stdout cannot be closed');
 E('ERR_STREAM_WRAP', 'Stream has StringDecoder set or is in objectMode');
+E('ERR_TLS_CERT_ALTNAME_INVALID',
+  (reason) => {
+    return `Hostname/IP does not match certificate's altnames: ${reason}`;
+  });
```

Comment:
You do not have to use a function here. Instead we can rely on util.format by replacing the function with:
`'Hostname/IP does not match certificate's altnames: %s'`.

This is done in all other places and it would be nice to keep it consistent.